weave/integrations/litellm/litellm_test.py

```diff
@@ -11,12 +11,12 @@
 
 # This PR:
 # https://github.com/BerriAI/litellm/commit/fe2aa706e8ff4edbcd109897e5da6b83ef6ad693
-# Changed the output format for OpenAI to use APIResponse when using async.
-# We should fix support for this, but for now we will just skip the test
-# parts that are affected by this change to unblock CI
-USES_RAW_OPENAI_RESPONSE_IN_ASYNC = (
-    semver.compare(litellm._version.version, "1.42.11") > 0
-)
+# Changed the output format for OpenAI to use APIResponse.
+# We can handle this in non-streaming mode, but in streaming mode, we
+# have no way of correctly capturing the output and not messing up the
+# users' code (that i can see). In these cases, model cost is not captured.
+
+USES_RAW_OPENAI_RESPONSE = semver.compare(litellm._version.version, "1.42.11") > 0
```

Comment:
better to use `importlib.metadata` here I think

Reply:
ok, changed